sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingAggregationSuite.scala

```diff
@@ -226,7 +226,7 @@ class StreamingAggregationSuite extends StateStoreMetricsTest with Assertions {
       AssertOnQuery { _.stateOperatorProgresses.head.numRowsTotal === 1 },
       AssertOnQuery { _.lastExecutedBatch.sink.numOutputRows == 0 },
       AddData(inputData, 10, 12, 14),
-      AdvanceManualClock(1000L), // watermark = 5, runs with the just added data
+      AdvanceManualClock(1000L), // watermark = 0, runs with the just added data
```

Comment:
Let's just remove watermark here in comment as you've done with further `AdvanceManualClock`